src/dialogs/more-info/controls/more-info-cover.js

```diff
@@ -64,6 +64,7 @@ class MoreInfoCover extends LocalizeMixin(PolymerElement) {
           </ha-labeled-slider>
         </div>
       </div>
+      <ha-attributes state-obj="[[stateObj]]"></ha-attributes>
```

Comment:
We should filter `current_position` (and others) that are already displayed in the controls above.